skbio/tree/tests/test_tree.py

```diff
@@ -276,6 +276,35 @@ def test_lowest_common_ancestor(self):
         with self.assertRaises(ValueError):
             t1.lowest_common_ancestor([])
 
+    def test_path(self):
+        """List of TreeNode object names in path between nodes"""
+        t1 = TreeNode.read(["((a,(b,c)d)e,f,(g,h)i)j;"])
+        t2 = t1.copy()
+        t3 = t1.copy()
+        init1 = t1.find("a") # tip
+        init2 = t2.find("j") # root node
+        init3 = t3.find("e") # internal node
+        fin1 = t1.find("g") # tip
+        fin2 = t2.find("c") # tip
+        fin3 = t3.find("i") # internal node
+        exp1 = ['e', 'j', 'i']
+        exp2 = ['j', 'e', 'd']
```

Comment:
This result is incorrect -- since `include_ends` is not set, the initial node "j" should not be in the path.